install/install.sh

```diff
@@ -98,6 +99,84 @@ install_dashboard(){
     -e SODA_GRAFANA_PORT=3000 \
     -e STORAGE_SERVICE_PLAN_ENABLED=true sodafoundation/dashboard:$DASHBOARD_RELEASE_VERSION  > /dev/null
 }
+
+install_srm_toolchain() {
+    install $@
+    mkdir -p /opt/srm-toolchain
+    srm_toolchain_work_dir=/opt/srm-toolchain
+    cp -a ./conf/srm-toolchain/. $srm_toolchain_work_dir
+    
+    prometheus_image_tag="${PROMETHEUS_IMAGE_TAG:=v2.23.0}"
+    prometheus_port="${SODA_PROMETHEUS_PORT:=9090}"
+    alertmanager_image_tag="${ALERTMANAGER_IMAGE_TAG:=v0.21.0}"
+    alertmanager_port="${SODA_ALERTMANAGER_PORT:=9093}"
+    grafana_image_tag="${GRAFANA_IMAGE_TAG:=7.3.5}"
+    grafana_port="${SODA_GRAFANA_PORT:=3000}"   
+        
+    # replace the Prometheus, Alertmanager and Grafana image tags and ports in the docker compose file.
+    sed -i 's/HOST_IP=.*/HOST_IP='"$HOST_IP"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/prom\/prometheus:.*/prom\/prometheus:'"$prometheus_image_tag"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/SODA_PROMETHEUS_PORT*/'"$prometheus_port"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/prom\/alertmanager.*/prom\/alertmanager:'"$alertmanager_image_tag"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/SODA_ALERTMANAGER_PORT*/'"$alertmanager_port"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/grafana\/grafana.*/grafana\/grafana:'"$grafana_image_tag"'/g' $srm_toolchain_work_dir/docker-compose.yml
+    sed -i 's/SODA_GRAFANA_PORT*/'"$grafana_port"'/g' $srm_toolchain_work_dir/docker-compose.yml
+
+    # replace host_ip and Alertmanager Port in prometheus configuration
+    sed -i 's/HOST_IP/'"$HOST_IP"'/g' $srm_toolchain_work_dir/prometheus/prometheus.yml
+    sed -i 's/SODA_ALERTMANAGER_PORT*/'"$alertmanager_port"'/g' $srm_toolchain_work_dir/prometheus/prometheus.yml
+
+    # replace host_ip and prometheus port in datasource configuration for grafana
+    sed -i 's/HOST_IP/'"$HOST_IP"'/g' $srm_toolchain_work_dir/grafana/provisioning/datasources/all.yml
+    sed -i 's/SODA_PROMETHEUS_PORT*/'"$prometheus_port"'/g' $srm_toolchain_work_dir/grafana/provisioning/datasources/all.yml
+    
+    # set PROMETHEUS as the default exporter
+    export PROMETHEUS=True
+
+    docker-compose -f $srm_toolchain_work_dir/docker-compose.yml up -d
```

Comment:
Use latest version of docker

```suggestion
    docker compose -f $srm_toolchain_work_dir/docker-compose.yml up -d
```

Reply:
This is the docker-compose command to bring up the SRM toolchain. Using the latest version of docker-compose supported by 20.04 (v1.29). The command is still `docker-compose` and not `docker compose.`